packages/css-modules-flow-types-loader/index.js

```diff
@@ -3,12 +3,52 @@
 import fs from 'fs';
 import printFlowDefinition from 'css-modules-flow-types-printer';
 
+/**
```

Comment:
I would like you to get rid of unnecessary comments in the code here. Some comments are valuable if the code is hard to read. But for a simple project like this I believe that

> The best comment is a good name for a method or class. 

And often good test cases is the way to go. 

Inspiration: https://refactoring.guru/smells/comments

Reply:
I'm guide by two basic rules/habits in this regard:
- the conviction that the code is for people and so giving the "what's the point" comment in addition to the "good name" rule is a good habit.
- jsdoc is some sort of a standard that is pretty well supported in all jetbrains IDEs and here on github

All that said, this arguably does get a bit verbose and I'm in no way inclined for enforcing this on anyone outside my project, so by all means strip those away ... or I can :+1:

Reply:
👍